hazelcast/src/main/java/com/hazelcast/spi/impl/operationexecutor/impl/OperationThread.java

```diff
@@ -58,11 +58,14 @@
 @ExcludedMetricTargets(MANAGEMENT_CENTER)
 public abstract class OperationThread extends HazelcastManagedThread implements StaticMetricsProvider {
 
-    final int threadId;
-    final OperationQueue queue;
+    int threadId;
+    OperationQueue queue;
```

Comment:
I think this is a dangerous change. We can't just make these into non final. Who synchronizes between thread creation and usage of this fields? We are prone to NPE if there isn't any.